packages/entities/entities-shared/src/components/entity-base-config-card/EntityBaseConfigCard.vue

```diff
@@ -187,6 +278,29 @@ const { axiosInstance } = composables.useAxios({
   headers: props.config?.requestHeaders,
 })
 
+const configFormatItems = [
+  {
+    label: 'Structured',
```

Comment:
Can `'Structured'` be internationalized? `'JSON'` and `'YAML'` are fine because I think they are universal in all languages.